pages/common/stdbuf.md

```diff
@@ -0,0 +1,15 @@
+# stdbuf
+
+> Run `command`, with modified buffering operations for its standard streams.
+
+- Change `stdin` buffer size to 512 kiB:
```

Comment:
Change from backticks to double quotes for all these cases.

Reply:
:heavy_check_mark: